k-distribution/src/main/scripts/lib/checkJava

```diff
@@ -46,15 +46,20 @@ else
   fi
 fi
 
-
+# The Java version information we're parsing here looks something like this:
+#
+#   openjdk 21.0.2 2024-01-16
```

Comment:
Previously, `java -version` would - for some implementations - produce something like:
```
openjdk version "21.0.2" 2024-01-16
```
on the first line.